baybe/constraints/conditions.py

```diff
@@ -41,15 +42,36 @@ def _is_not_close(x: ArrayLike, y: ArrayLike, rtol: float, atol: float) -> np.nd
         given tolerances.
 
     """
-    return np.logical_not(np.isclose(x, y, rtol=rtol, atol=atol))
+    return np.logical_not(isclose(x, y, rtol=rtol, atol=atol))
+
+
+def isclose(x: ArrayLike, y: ArrayLike, rtol: float, atol: float) -> np.ndarray:
+    """Return a boolean array indicating where ``x`` and ``y`` are close.
+
+    The equivalent to ``numpy.isclose``.
+    Using ``np.isclose`` alongside Polars dataframes results in this error:
+    ``TypeError: ufunc 'isfinite' not supported for the input types``.
+
+    Args:
+        x: First input array to compare.
+        y: Second input array to compare.
+        rtol: The relative tolerance parameter.
+        atol: The absolute tolerance parameter.
+
+    Returns:
+        Returns a boolean array of where ``x`` and ``y`` are equal within the
+        given tolerances.
+
```

Comment:
Please also change in `_is_not_close`:

```suggestion
    Returns:
        A boolean array of where ``x`` and ``y`` are equal within the
        given tolerances.
```

Reply:
changed